QuickSetup.md

```diff
@@ -0,0 +1,45 @@
+# Mac setup
+
+### Cloning the project
+First, you'll need to clone the project on your machine.  
+
+Pommerman project is hosted on GitHub.  
+GitHub is a web-based hosting service for version control using git. You can create a free  
+account on GitHub, and clone the project to your local machine.
+
+GitHub url : https://github.com/MultiAgentLearning/playground
```

Comment:
They're already at this repository, so this section is all redundant.

Reply:
Done